csrc/src/cache_manager.rs

```diff
@@ -363,25 +363,25 @@ pub fn reshape_and_cache_flash_t<
     }
     if [num_blocks, block_size, num_heads, head_size] != key_cache.dims() {
         candle_core::bail!(
-            "Only support key_cache with shape [num_blocks, num_heads, head_size, block_size] (got {:?})",
+            "Only support key_cache with shape [{num_blocks}, {num_heads}, {head_size}, {block_size}] (got {:?})",
```

Comment:
It should be `[num_blocks, block_size, num_heads, head_size]`, right ?

Reply:
done